examples/foomedical/src/components/utils.tsx

```diff
@@ -0,0 +1,24 @@
+import { Resource } from '@medplum/fhirtypes';
+
+/**
+ * Cleans a resource of meta data including `lastUpdated`, `versionId`, and `author`. This allows the resource to be updated without
+ * retaining outdated information.
+ *
+ * @param resource - The resource to be cleaned
+ * @returns Resource - a resource without certain meta data. This allows the resource in question to be safely updated.
+ */
+export function cleanResource(resource: Resource): Resource {
```

Comment:
I don't think this should be necessary. By default, the server ignores any changes the client makes to these `meta` fields. @pamella please let me know if you were seeing different behavior

Reply:
Good to know. I removed the util, thanks!